modules/osdk-cli-reference-run.adoc

```diff
@@ -1,36 +1,46 @@
-[id="osdk-cli-reference-up_{context}"]
-= up
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
 
-The `operator-sdk up` command has subcommands that can launch the Operator in
-various ways.
+[id="osdk-cli-reference-run_{context}"]
+= run
 
-== local
+The `operator-sdk run` command provides options that can launch the Operator in
+various environments.
 
-The `local` subcommand launches the Operator on the local machine by building
-the Operator binary with the ability to access a Kubernetes cluster using a
-`kubeconfig` file.
-
-.`up local` arguments
+.`run` arguments
 [options="header",cols="1,3"]
 |===
 |Arguments |Description
 
 |`--kubeconfig` (string)
 |The file path to a Kubernetes configuration file. Defaults: `$HOME/.kube/config`
 
+|`--local`
+|The Operator is run locally by building the Operator binary with the ability to
+access a Kubernetes cluster using a `kubeconfig` file. Cannot be set with
+another run type flag.
```

Comment:
What's a "run type" flag? Should it be hyphenated?

Reply:
Removed it since only one type (`--local`) is being shown right now downstream.